karma.conf.js

```diff
@@ -70,6 +88,7 @@ module.exports = (config) => {
         },
         coverageIstanbulReporter: {
             fixWebpackSourcePaths: true,
+            skipFilesWithNoCoverage: true,
```

Comment:
@akhenry this may fix some of the code coverage inconsistency that we're seeing in our reports